docs/4.Running-the-Integration-Test-Suite.md

```diff
@@ -0,0 +1,4 @@
+## Running the localnet Integration Test Suite
+If one has docker and docker-compose installed on their machines,
+```./scripts/setup.sh```
+aught to spin up a localnet with ganache-cli, 4 validator nodes with peggy
```

Comment:
Running this script gives an error at the end of the docker build:

`=> exporting to image                                                                                                                                                     0.3s
 => => exporting layers                                                                                                                                                    0.3s
 => => writing image sha256:fec84c8f14c6a97d6c4559e6b7a5045a537bdbd1365e8f6ad62cda222ee3ec36                                                                               0.0s
 => => naming to docker.io/sifchain/sifnoded:localnet                                                                                                                      0.0s
ERROR: The Compose file '././genesis/docker-compose.yml' is invalid because:
services.sifnode2.networks.network_mode contains an invalid type, it should be an object, or a null
services.sifnode4.networks.network_mode contains an invalid type, it should be an object, or a null
======================
If one chooses the force kill remember to docker stop $(docker ps -aq) to spin down ganache-cli
======================`

Reply:
Reading into this guy I realize I created a circular dependency by neglecting to prune the genesis_sifchain network, fixing now